api/models.py

```diff
@@ -1841,6 +1841,12 @@ class OrgTypes(models.TextChoices):
     phone_number = models.CharField(verbose_name=_("phone number"), blank=True, null=True, max_length=100)
     last_frontend_login = models.DateTimeField(verbose_name=_("last frontend login"), null=True, blank=True)
     accepted_montandon_license_terms = models.BooleanField(verbose_name=_("has accepted montandon license terms?"), default=False)
+    limit_access_to_guest = models.BooleanField(
+        help_text="User having this value as true explictly is treated as guest user.Despite of having all the \
+        permission, if this value is set to true, the user is deprived of all the non-guest user permission.",
+        verbose_name=_("limit access to guest user permissions"),
+        default=True,
+    )
```

Comment:
@thenav56 Modifying it creates a new migration file. Is it ok to create new migration file or merge it to the previous migrations.

Reply:
@Rup-Narayan-Rajbanshi 
Use the content from the new migration file to update the existing one. Since it's just the help_text changed, it will not require any database change.

Reply:
@thenav56 Done.

Reply:
@Rup-Narayan-Rajbanshi It's not done. Can you re-check?

Reply:
@thenav56 I think its done now.